.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: CI
+
+on:
+  pull_request:
+
+env:
+  node_version: 20
+  pnpm_version: 9
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
these actions need to also be updated to their latest as well, though, there are too many for me to do an individual suggestion for 😄

Reply:
Done!